test/WebJobs.Script.Tests/Workers/Rpc/RpcFunctionInvocationDispatcherTests.cs

```diff
@@ -204,6 +204,26 @@ public async Task FunctionDispatcherState_Transitions_From_Starting_To_Initializ
             await WaitForFunctionDispactherStateInitialized(functionDispatcher);
         }
 
+        [Fact]
+        public async Task FunctionDispatcherState_Transitions_From_Starting_To_Initialized_To_Disposing()
+        {
+            RpcFunctionInvocationDispatcher functionDispatcher = GetTestFunctionDispatcher();
+            FunctionMetadata func1 = new FunctionMetadata()
+            {
+                Name = "func1",
+                Language = "node"
+            };
+            var functions = new List<FunctionMetadata>()
+            {
+                func1
+            };
+            await functionDispatcher.InitializeAsync(functions);
+            Assert.True(functionDispatcher.State == FunctionInvocationDispatcherState.Initializing || functionDispatcher.State == FunctionInvocationDispatcherState.Initialized);
```

Comment:
I did this to avoid a flaky test. Since the state changing from Initializing => Initialized is done asynchronously in the background, there's a chance that it could return as "Initialized" before our intentional delay, if it completes "too soon"

Reply:
TestWorkerChannel's implementation of process start has a delay. See here: https://github.com/Azure/azure-functions-host/blob/dev/test/WebJobs.Script.Tests/Workers/Rpc/TestRpcWorkerChannel.cs#L74 You can increase the delay if needed!

Reply:
Perfect! Done!!